go.mod

```diff
@@ -114,4 +113,22 @@ require (
 	gopkg.in/validator.v2 v2.0.0-20210331031555-b37d688a7fb0
 	gopkg.in/yaml.v2 v2.4.0
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
+	modernc.org/sqlite v1.13.1
+)
+
+require (
```

Comment:
Please join `require` sections in one.

Reply:
This is new in Go 1.17: https://golang.org/doc/go1.17#go-command

> Because the number of explicit requirements may be substantially larger in an expanded Go 1.17 go.mod file, the newly-added requirements on indirect dependencies in a go 1.17 module are maintained in a separate require block from the block containing direct dependencies.

Reply:
Oh, great! I didn't know about that. Let's change it everywhere then.

Reply:
#1974.